forks/Microsoft.Health.FellowOakDicom/Serialization/Utf8JsonReaderExtensions.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2012-2021 fo-dicom contributors.
+// Licensed under the Microsoft Public License (MS-PL).
+
+using System.Text.Json;
+
+namespace Microsoft.Health.FellowOakDicom.Serialization
+{
+    internal static class Utf8JsonReaderExtensions
+    {
+        public static void Assume(this ref Utf8JsonReader reader, JsonTokenType tokenType)
```

Comment:
Do you need this file? Is this from Fo.Dicom too?

Reply:
Yes. Otherwise the converter code doesn't compile. Not sure how this is not in the published fo-dicom 5.0.3 version.